### PR TITLE
Adde a timeout to show a info popup

### PR DESCRIPTION
**Pull request by alexjavabraz**

### CODE REVIEW

File: src/components/liquality/ConfirmLiqualityTransaction.vue [hunk 1]
```diff
@@ -203,23 +216,46 @@ export default class ConfirmLiqualityTransaction extends Vue {
...
+          error.message = 'Liquality is closed or Account is not selected';
+          throw error;
+        })
+        .catch((err) => {
+          this.confirmTxState.send('error');
+          txError = err.message;
+        });
+      return [txError, this.txId];
+    } catch (e) {
+      const error = new LiqualityError();
```
Comment (josedahlquist) [inline src/components/liquality/ConfirmLiqualityTransaction.vue:255]: why are not using the parameterized exception?

File: src/services/LiqualityService.ts [hunk 2]
```diff
@@ -53,9 +55,15 @@ export default class LiqualityService extends WalletService {
 
   // eslint-disable-next-line class-methods-use-this
   async isConnected(): Promise<boolean> {
+    let continueCounting = true;
     return new Promise<boolean>((resolve, reject) => {
-      const walletAddresses: WalletAddress[] = [];
+      setTimeout(() => {
+        if (continueCounting) {
+          reject(new LiqualityError());
```
Comment (josedahlquist) [inline src/services/LiqualityService.ts:62]: I think you could resolve with false instead
```suggestion
          resolve(false);
```

File: src/services/LiqualityService.ts [hunk 3]
```diff
@@ -66,19 +74,10 @@ export default class LiqualityService extends WalletService {
...
-                serializedPath: liqualityAddress.derivationPath,
-                publicKey: liqualityAddress.publicKey,
-                path: [0],
-              });
-            });
+        .then(() => {
           resolve(true);
-        }).catch(reject);
+        })
+        .catch(reject);
```
Comment (josedahlquist) [inline src/services/LiqualityService.ts:80]: same here:
```suggestion
        .catch(() => resolve(false));
```

File: src/components/create/SendBitcoin.vue [hunk 4]
```diff
@@ -171,10 +175,14 @@ export default class SendBitcoin extends Vue {
...
   @Emit()
-  toTrackingId([txError, txId]: string[]) {
+  toTrackingId([txError, txId, urlToMoreInformation, errorType, messageToUserOnLink]: string[]) {
     if (txError !== '') {
       this.txError = txError;
       this.showTxErrorDialog = true;
+      this.urlToMoreInformation = urlToMoreInformation;
+      this.errorType = errorType;
+      this.messageToUserOnLink = messageToUserOnLink;
+      this.showTxErrorDialog = true;
```
Comment (josedahlquist) [inline src/components/create/SendBitcoin.vue:185]: showTxErrorDialog is already set a few lines above
```suggestion
```

File: src/components/liquality/ConfirmLiqualityTransaction.vue [hunk 5]
```diff
@@ -203,23 +216,46 @@ export default class ConfirmLiqualityTransaction extends Vue {
...
   environmentContext = EnvironmentContextProviderService.getEnvironmentContext();
 
+  // eslint-disable-next-line class-methods-use-this
+  @Emit('successConfirmation')
+  errorOnConnection() {
+    this.confirmTxState.send('error');
+    const error = new LiqualityError();
+    return [error.message, this.txId, error.urlToMoreInformation,
+      error.errorType, error.messageToUserOnLink];
+  }
```
Comment (josedahlquist) [inline src/components/liquality/ConfirmLiqualityTransaction.vue:226]: I don't get this, you are instantiating an empty LiqualityError and returning its part... is this truly needed?

File: src/services/LiqualityService.ts (hunk 2 above)
Comment (alexjavabraz) [reply src/services/LiqualityService.ts:62]: If the isConnected method resolves false, then we will try to reconnect, if the user disables the Liquality the thread is abandoned instead we´re throwing this exception.

**Pull request closed without merging**